lib/plugins/acts_as_customizable/lib/acts_as_customizable.rb

```diff
@@ -155,7 +155,7 @@ def respond_to?(method, include_private = false)
         def for_custom_field_accessor(method_symbol)
           match = /\Acustom_field_(?<id>\d+)=?\z/.match(method_symbol.to_s)
           if match
-            custom_field = CustomField.find_by_id(match[:id])
+            custom_field = available_custom_fields.find { |cf| cf.id.to_s == match[:id] }
```

Comment:
@NobodysNightmare Not complaining about the style but about performance (and style will no longer be an issue. 
`available_custom_fields` returns a scope. Your implementation will load all the custom fields and the pick the one you actually need. This is more performant when more than one custom_field is to be accessed. Is that your expectation? Otherwise you should use:
`available_custom_fields.find_by_id(match[:id])`